build/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 Target Brands, Inc. All rights reserved.
+#
+# Use of this source code is governed by the LICENSE file in this repository.
+
+###########################################################################
+##    docker build --no-cache -t vela-npm:local -f build/Dockerfile .    ##
+###########################################################################
+
```

Comment:
Not a big deal, but it looks like every other repo keeps the Dockerfile at the top level

Reply:
Moved it tot he root to keep it consistent